builtins/amp-img.md

```diff
@@ -109,14 +109,22 @@ This attribute is similar to the `src` attribute on the `img` tag. The value mus
 
 Same as `srcset` attribute on the `img` tag. For browsers that do not support `srcset`, `<amp-img>` will default to using `src`. If only `srcset` and no `src` is provided, the first url in the `srcset` will be selected.
 
-**sizes**
+**sizes (optional)**
+
+This attribute is optional for `<amp-img>`. If the `srcset` attribute is provided, `<amp-img>` will autogenerate a `sizes` attribute for the underlying `<img>` when none is specified. If the `srcset` attribute is not provided, `sizes` does nothing. If you do wish to set the `sizes` attribute on `<amp-img>`, it will be propagated to the underlying `<img>` tag with the following side-effects:
```

Comment:
I'm a bit confused here, Does `sizes` not do the default AMP sizes behavior if you do not specify `srcset`? When it says `sizes` does nothing, that is what I think.

Do you mean to say that `sizes` has no effect on the image src itself when `srcset` is not specified?

Reply:
Both. 
1. In normal html on `<img>`, if you specify `sizes` without `srcset` nothing happens. 
2. In `<amp-img>` if you specify `sizes` without `srcset`, it's propagated to the underlying `<img>`, but that's useless because browsers do nothing. 
3. If you don't specify the `srcset` attribute AND don't specify the `sizes` attribute, we don't autogenerate a `sizes` attribute.